go/porcelain/deploy_test.go

```diff
@@ -65,3 +76,29 @@ func TestAddWithLargeMedia(t *testing.T) {
 		t.Fatalf("expected `%v`, got `%v`", "sum3:originalsha", out2)
 	}
 }
+
+func TestOpenAPIClientWithWeirdResponse(t *testing.T) {
+	server := httptest.NewServer(http.HandlerFunc(func(rw http.ResponseWriter, req *http.Request) {
+		rw.Header().Set("Content-Type", "application/json; charset=utf-8")
+		rw.WriteHeader(408)
+		rw.Write([]byte(`{ "foo": "bar", "message": "a message", "code": 408 }`))
+	}))
+	defer server.Close()
+
+	httpClient := http.DefaultClient
+	authInfo := runtime.ClientAuthInfoWriterFunc(func(r runtime.ClientRequest, _ strfmt.Registry) error {
+		_ = r.SetHeaderParam("User-Agent", "buildbot")
```

Comment:
kinda nitpicking but is this `_ =` part needed?

Reply:
Good question.

Reply:
Nope! Good catch.